automatic/alldup/update.ps1

```diff
@@ -12,16 +12,21 @@ function global:au_SearchReplace {
   }
 }
 
-function global:au_BeforeUpdate { Get-RemoteFiles -Purge -NoSuffix }
+function global:au_BeforeUpdate {
+  Get-RemoteFiles -Purge -NoSuffix
+}
 
 function global:au_GetLatest {
-  $download_page = Invoke-WebRequest -Uri $releases
+  $downloadPage = Invoke-WebRequest -Uri $releases -UseBasicParsing
 
-  $re = 'alldup.*\.exe$'
-  $url     = $download_page.links | ? href -match $re | select -First 1 -expand href
-  $version = $download_page.links | ? href -match "alldup_version\.php$" | select -first 1 -expand innerText
+  $re = 'AllDupSetup.exe$'
+  $url = $downloadPage.links | Where-Object href -match $re | Select-Object -First 1 -expand href
+  $downloadPage.links | Where-Object outerHTML -Match 'alldup_version\.php"\>[\d\.]+<' | Select-Object -First 1 | ForEach-Object { $_ -match '\>(?<version>[\d\.]+)<' }
 
-  @{ URL32 = $url; Version = $version }
+  @{
+    URL32 = $url
+    Version = $matches.version
```

Comment:
```suggestion
    Version = $Matches.version
```

We have commonly used an uppercase M when using the Matches variables.

Reply:
That's very strange. `$matches` is just another variable and variables traditionally start with a lower case character. Why would you need to use an uppercase letter?

Reply:
We don't need it from a code standpoint. It is for consistency to keep the casing similar to what we use elsewhere.

Reply:
On this: $Matches is a variable being defined by PowerShell, so the approach I've generally seen uses that - similar to guidance around foreach vs ForEach-Object.

Reply:
If we're going to keep going back on forth on the styling changes that have been requested, this PR is going to go nowhere.

@JPRuskin can you _please_ make the changes requested, so we can merge this PR?